stores/user_store.jsx

```diff
@@ -459,6 +459,22 @@ class UserStoreClass extends EventEmitter {
         return this.getStatuses()[id] || UserStatuses.OFFLINE;
     }
 
+    getTimezone(id) {
```

Comment:
Could you turn this into a selector instead? Preferably in mattermost-redux since I figure the mobile apps are going to need this as well

Then the timezone can get passed in as a prop to the components that need it instead of importing UserStore (which we're trying to reduce usage of so we can remove it)

Reply:
It would definitely make sense as a selector. Something like this would be nice and simple.
```
const getCurrentTimezone = createSelector(
    getCurrentUser,
    (currentUser) => {
        ...
    }
);
```

If we ever care about the timezone of another user, we could add one that's more detailed.

Reply:
👍